packages/import-service/.env

```diff
@@ -1,4 +1,4 @@
 IMPORT_S3_BUCKET=nodejs-aws-task5-csv
```

Comment:
Even this constants could be exposed, better to not commit .env file to a repo

Reply:
They have a bit different rules:

> Note: .env, .env.development, and .env.production files should be included in your repository as they define defaults. .env*.local should be added to .gitignore, as those files are intended to be ignored. .env.local is where secrets can be stored.

https://www.serverless.com/plugins/serverless-dotenv-plugin

So, I'm storing confidential data in .env.*.local files.

Reply:
Ok